README.md

```diff
@@ -16,14 +18,21 @@
 Файл с весами и конфигурацией обученной модели _bird_recognition.h5_ будет установлен автоматически при первом использовании bird_predictor.py.
 Датасеты, использованные при обучении и тестировании модели, можно скачать с [Google Drive](https://drive.google.com/drive/folders/1ZDlgOtVB-Jdkqt-cQrEdyl-QMCJwF2kq?usp=sharing).
 
+#### 2 вариант: Используя расширение Dev Containers для VSC
+
+1. Скачать и распаковать архив с файлами репозитория в локальный каталог.
```

Comment:
Let's leave it more like "clone the repository with a 

'''bash
git clone _git-repo-url_
'''
"